src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useMemo, useState } from 'react';
+import React, { ReactElement, useEffect, useMemo, useState } from 'react';
```

Comment:
Could we add groups to the Custom repositories modal? The first column could be Packages/groups and then just a name of the group.

![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/01a04fbb-107b-4148-a2ec-6fae21593de9)